micro-rdk-server/esp32/main.rs

```diff
@@ -34,7 +34,11 @@ mod esp32 {
 
         micro_rdk::esp32::esp_idf_svc::log::EspLogger::initialize_default();
 
-        let repr = RobotRepresentation::WithRegistry(Box::<ComponentRegistry>::default());
+        let mut r = Box::<ComponentRegistry>::default();
+        if let Err(e) = micro_rdk_modular_driver_example::register_models(&mut r) {
```

Comment:
- This is going to register the water pump and moisture sensor too. Is that intended? f you just want to register just the heap and rssi sensors, then you I think will need to make their individual registration functions `pub fn` in `micro-rdk-modular-driver-example` so you can call them directly here.
- Should this be gated on `builtin-components` somehow? Though I suppose that is a `micro-rdk` feature not a `micro-rdk-server` feature.

Reply:
- changed to just heap sensor
- I see what you mean. I don't feel strongly about it either way. I know the intention is to have this available out of the box for users so they can see immediately how app works.